### PR TITLE
In case of error, add default value to getters

### PR DESCRIPTION
**Pull request by cmeldas**
When key is not found, you can provide default value to return for getInt and getFloat.

### CODE REVIEW

File: src/ArduinoNvs.cpp [hunk 1]
```diff
@@ -166,7 +166,7 @@ bool ArduinoNvs::setBlob(String key, std::vector<uint8_t> &blob, bool forceCommi
   return setBlob(key, &blob[0], blob.size(), forceCommit);
 }
 
-int64_t ArduinoNvs::getInt(String key)
+int64_t ArduinoNvs::getInt(String key, int32_t default_value)
```
Comment (rpolitex) [inline src/ArduinoNvs.cpp:169]: Hi!
this function could return `int64_t`, so a `def_value` should be at least `int64_t`

**Review by rpolitex — COMMENTED** [review]
Hi, 
This proposed changes are great! 

Thank you for your contribution!

**Comment by rpolitex** [comment]
Released in v.2.7

File: src/ArduinoNvs.cpp (hunk 1 above)
Comment (cmeldas) [reply src/ArduinoNvs.cpp:169]: ok, thanks for correction

**Pull request merged**